tests/ocpp-server/server.py

```diff
@@ -103,6 +110,47 @@ async def send_clear_cache(self):
         else:
             logging.info("%s failed", Action.ClearCache)
 
+    async def send_get_base_report(self):
+        logging.info("Executing send_get_base_report...")
+        request = ocpp.v201.call.GetBaseReport(reportBase=ReportBaseType.ConfigurationInventory)  # Use correct ReportBaseType
+        try:
+            response = await self.call(request)
+            logging.info("Send GetBaseReport")
+
+            if response.status == "Accepted":  # Adjust depending on the structure of your response
+                logging.info("Send GetBaseReport successful")
+            else:
+                logging.info("Send GetBaseReport failed")
+        except Exception as e:
+            logging.error(f"Send GetBaseReport failed: {str(e)}")
+        logging.info("send_get_base_report done.")
+
+# Define argument parser
+parser = argparse.ArgumentParser(description='OCPP Charge Point Simulator')
+parser.add_argument('--request', type=str, help='OCPP 2 Command Name')
+parser.add_argument('--delay', type=int, help='Delay in seconds')
+parser.add_argument('--period', type=int, help='Period in seconds')
+
+args = parser.parse_args()
+
+# Function to send OCPP command
+async def send_ocpp_command(cp, command_name, delay=None, period=None):
+    # If delay is not None, sleep for delay seconds
+    if delay:
+        await asyncio.sleep(delay)
+
+    # If period is not None, send command repeatedly with period interval
+    if period:
+        while True:
+            if command_name == 'GetBaseReport':
```

Comment:
Please 

- use existing ocpp library data structure
- use a `match ... case` code flow for the 2 commands: ClearCache and GetBaseReport
- implement the 2 commands support

Reply:
`try:
                    match command_name:
                        case 'ClearCache':
                            logging.info("ClearCache parser working")
                            await cp.send_clear_cache()
                        case 'GetBaseReport':
                            logging.info("GetBaseReport parser working")
                            await cp.send_get_base_report()
                        case _:
                            logging.warning(f"Unsupported command {command_name}")`